assignments/this.js

```diff
@@ -1,26 +1,60 @@
 /* The for principles of "this";
 * in your own words. explain the four principle for the "this" keyword below.
 *
-* 1. 
-* 2. 
-* 3. 
-* 4. 
+* 1. Implicit Binding = this is automatic, this will happen no matter what, it's simple/direct
+* 2. Explicit = 
+* 3. Window = trys to this keyord from window, not useable
+* 4. New = Takes a function call and binds it to a new function
```

Comment:
`new` allows us to make a new instance of an object using a constructor function.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/new